app/models/mongo_fulfillment.rb

```diff
@@ -0,0 +1,36 @@
+# Indicator that a pledge by the primary fund was cashed in,
+# which in turn indicates that the patient used our pledged money.
+class Fulfillment
```

Comment:
Throwing an error on `rails migrate_to_pg:clinic_user_patient` because of this!

```suggestion
class MongoFulfillment
```